### PR TITLE
chore: fix CI

### PR DESCRIPTION
**Pull request by JoviDeCroock**
This is a temp placeholder, so I can see what holds the CI up in our webview.

Caching etc seems covered now, could someone double check if there's any regression in things covered by the CI, I think not but yes I'm not that experienced with this repo yet.

### CODE REVIEW

**Comment by JoviDeCroock** [comment]
I would also propose to reduce the filesize checks to more reasonable amounts:

``` PASS  ./packages/apollo-link/lib/bundle.min.js: 1.66KB < maxSize 3KB (gzip) 

 PASS  ./packages/apollo-link-batch/lib/bundle.min.js: 1.44KB < maxSize 1.5KB (gzip) 

 PASS  ./packages/apollo-link-batch-http/lib/bundle.min.js: 5.66KB < maxSize 6.8KB (gzip) 

 PASS  ./packages/apollo-link-dedup/lib/bundle.min.js: 956B < maxSize 3KB (gzip) 

 PASS  ./packages/apollo-link-error/lib/bundle.min.js: 895B < maxSize 1KB (gzip) 

 PASS  ./packages/apollo-link-http/lib/bundle.min.js: 5.13KB < maxSize 6KB (gzip) 

 PASS  ./packages/apollo-link-polling/lib/bundle.min.js: 790B < maxSize 1KB (gzip) 

 PASS  ./packages/apollo-link-retry/lib/bundle.min.js: 2.07KB < maxSize 2.5KB (gzip) 

 PASS  ./packages/apollo-link-ws/lib/bundle.min.js: 8.75KB < maxSize 10KB (gzip) ```

**Comment by JoviDeCroock** [comment]
Weird, the circleci is not starting again....

**Comment by hwillson** [comment]
I think it will @JoviDeCroock - it's just being delayed by other queued up and running tests (related to other projects).

**Comment by JoviDeCroock** [comment]
> I think it will @JoviDeCroock - it's just being delayed by other queued up and running tests (related to other projects).

Allright, being too hasty :D was trying to debug the issue but not really finding the queue issue atm

**Comment by JoviDeCroock** [comment]
Shouldn't we add monorepo as required aswell?

**Comment by JoviDeCroock** [comment]
Allright, thank you for the help!

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,277 +1,85 @@
...
   Prettier Check:
-    docker: [ { image: 'circleci/node:8' } ]
+    <<: *defaults
     steps:
-      - attach_workspace:
-          at: .
+      - <<: *attach
       - run: npm run lint-check
 
   Docs:
```
Comment (hwillson) [inline .circleci/config.yml]: I know we're building the docs like this in other repos, but I have a feeling this isn't really necessary. Our Netlify based docs build and deploy process is already taking care of this, and if the docs fail to build for any reason, an  error will be reported by the Netlify GitHub check. Let's leave this docs CI step for now, but food for future thought.

**Pull request merged**